src/test/resources/README.md

```diff
@@ -0,0 +1 @@
+test resources go here
```

Comment:
We can add a bunch of test files for core functionality (maybe in a large source directory tracked with git-lfs) in a different PR: a matched SAM/BAM/FASTA/VCF might be ideal. In that case, do we really need a stub file?